superset/utils/cache_manager.py

```diff
@@ -28,41 +34,47 @@ def __init__(self) -> None:
         self._filter_state_cache = Cache()
         self._explore_form_data_cache = Cache()
 
+    @staticmethod
+    def _init_cache(
+        app: Flask, cache: Cache, cache_config_key: str, required: bool = False
+    ) -> None:
+        cache_config = app.config[cache_config_key]
+        cache_type = cache_config.get("CACHE_TYPE")
+        if app.debug and cache_type is None:
+            cache_threshold = cache_config.get("CACHE_THRESHOLD", math.inf)
+            cache_config.update(
+                {"CACHE_TYPE": "SimpleCache", "CACHE_THRESHOLD": cache_threshold,}
```

Comment:
```suggestion
                {"CACHE_TYPE": "SimpleCache", "CACHE_THRESHOLD": cache_threshold}
```

Reply:
I'm surprised this is not catched by `black`.

Reply:
Yeah this has been annoying me for a long time. I'm sure it's been fixed in a new version of black, would be nice to take a stab at updating it.